### DISCUSSION TITLE
Shrink width / height to change image proportions possible?

### DISCUSSION THREAD

**Discussion by ilia239** [mikebrady/shairport-sync#1909] [General]
Hello,
I wonder whether it is possible to adjust width/height proportions of an image before streaming it from the device? 
The problem I am trying to solve - a picture of a perfect square streamed to my Epson streamer looks like a rectangular, the width is slightly longer than length, proportions are not correct . As I am using the setup for drawing, even small disproportions are not acceptable. I am looking for a way to fix width/height proportions of the final image on the screen. 
My Epson streamer does not have an option to shrink width or height, which would be most convenient solution indeed, so I need a some kind of pre-processing: to scale the 1080 x 720 image to i.e. 1055 x 720 and fill the left and right space with black
How can I do this with Shairport? What other options I have?
regards,
ilia

**Comment by mikebrady** [comment]
Hi there. I’m afraid Shairport Sync doesn’t stream images at all. It only streams audio.

**Reply by ilia239** [reply]
Ok, thanks for this info anyways.
